nba/config.py

```diff
@@ -1 +1,11 @@
-# TODO: read in your yaml file and parse it using the data classes defined in dataclasses.py
+import yaml
+from pydantic import BaseModel
+from config_classes import DataPaths
```

Comment:
Hey Pushpin @ps2016-mt , 
I think we might have a mismatch with config.yaml?
As config.yaml has data and model, config_classes should have  class Data and class Model, in config.py we should have 'from config_classes import Data, Model'